tests/integration/components/bs-accordion-test.js

```diff
@@ -46,8 +64,9 @@ module('Integration | Component | bs-accordion', function (hooks) {
     `);
 
     assert
-      .dom(`.${accordionItemClass()}:first-child .${accordionItemHeadClass()}`)
-      .hasNoClass('collapsed', `${accordionItemHeadClass()} has not collapsed class`);
+      .dom(`.${accordionItemClass()}:first-child ${accordionItemClickableSelector()}`)
+      .hasNoClass('collapsed', `${accordionItemHeadClass()} has not collapsed class`)
+      .hasAria('expanded', 'true', 'aria: is expanded');
```

Comment:
This test (and some others in this file) weren't actually testing what we expected. The `collapsed` class is set on the `button` not on `.accordion-header`. This wasn't noticed as we only tested for absence for a class. This is always risky as it may pass just because using a wrong selector.